keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/keel/clouddriver/model/ActiveServerGroup.kt

```diff
@@ -186,7 +186,7 @@ data class ActiveServerGroupImage(
   ) : this(
     imageId = imageId,
     appVersion = tags.getTag("appversion")?.substringBefore("/"),
-    baseImageVersion = tags.getTag("base_ami_version"),
+    baseImageName = description?.let{ Regex("""ancestor_name=([\w-]+)""").find(it)?.groupValues?.get(1) },
```

Comment:
As a reader, I'd really appreciate having an example of a `description` string here as a comment. 

There's a good example in `ActiveServerGroupTest`:

```
name=fnord, arch=x86_64, ancestor_name=xenialbase-x86_64-201811142132-ebs, ancestor_id=ami-0fe383ecea7f75eac, ancestor_version=nflx-base-5.308.0-h1044.b4b3f78
```

Reply:
Note also that the `get` is going to throw an exception if the description exists but doesn't match the regex. I'm not sure what behavior we actually want in that situation.

Reply:
That's what all the null-safety is for. If it doesn't match `find` will return `null` (and `baseImageName` will end up as `null`)

Reply:
Ah, thanks.